tardis/io/tests/test_model_reader.py

```diff
@@ -0,0 +1,35 @@
+import tardis
+from tardis.io.model_reader import (read_artis_density,
+read_simple_ascii_abundances)
+
```

Comment:
I'm not sure but if there is no necessity for brackets, remove them and put in a single line.